core/src/core_tests/workflow_tasks.rs

```diff
@@ -2054,3 +2054,36 @@ async fn continue_as_new_preserves_some_values() {
         .await
         .unwrap();
 }
+
+#[rstest]
+#[tokio::test]
+async fn ignorable_events_are_ok(#[values(true, false)] attribs_unset: bool) {
+    let mut t = TestHistoryBuilder::default();
+    t.add_by_type(EventType::WorkflowExecutionStarted);
+    let id = t.add_get_event_id(
+        EventType::Unspecified,
```

Comment:
Do event types that aren't in the compiled proto definitions end up as `Unspecified`?

Reply:
Yes